src/fondue/Fondue.js

```diff
@@ -269,4 +269,47 @@ export default class Fondue {
 			return "";
 		}
 	}
+
+	// Returns an array of all supported Unicode characters
+	// from the "best" cmap.
+	get supportedCharacters() {
+		const cmap = this.getBestCmap();
+		const chars = [];
```

Comment:
This variable is being mutated (`chars.push`). Making it `let` instead of `const` is a nice way to signal that.